.jenkins/pytorch/win-test.sh

```diff
@@ -51,7 +51,7 @@ run_tests() {
         if [[ "${JOB_BASE_NAME}" == *-test1 ]]; then
             export PYTORCH_COLLECT_COVERAGE=1
             $SCRIPT_HELPERS_DIR/test_python_nn.bat "$DETERMINE_FROM" && \
-            $SCRIPT_HELPERS_DIR/test_libtorch.bat
+            $SCRIPT_HELPERS_DIR/test_libtorch.bat && \
```

Comment:
nit: Also, with `set -ex`, we don't need this chain anymore. It looks a bit weird to chain sth with an if condition.

Reply:
good suggestion. updated